charts/kubetail/templates/service-account.yaml

```diff
@@ -1,7 +1,14 @@
+{{- if .Values.serviceAccount.create -}}
```

Comment:
A ServiceAccount is only needed for `authMode: cluster` so I think it would be simpler to remove the `serviceAccount.create` option and create it automatically based on the auth-mode.

Reply:
Each application should have his own service account. The usage of the default service account isn't recommend.

I would always create one. But I would add an toggle around the creation of RBAC.

Reply:
Some enterprise think it would be a great idea to manage service account centrally. In such scenario, I may have to use an existing serviceAccount...

A lot of charts are proving this toggle, too. For example the grafana

https://github.com/search?q=repo%3Agrafana%2Fhelm-charts%20%22serviceAccount.create%22&type=code

II would recommend to have an explicit toggle here.